src/pages/SetPassword.vue

```diff
@@ -18,9 +22,19 @@
         name="password"
         type="password"
       />
-      <p>
-        Please choose a personal password to encrypt your new account.
-      </p>
+      <ae-label
+        :for="_uid"
+        :help-text="errors.first('password_repeat')"
+        help-type="dramatic"
+      >Confirm your password</ae-label>
+      <ae-input
+        v-validate="'required|confirmed:'+password"
+        v-focus="true"
+        :id="_uid"
+        v-model="password_repeat"
```

Comment:
This value actually is not used, so don't need to bind `password_repeat` with this input.